### DISCUSSION TITLE
Negative value TAX. help please

### DISCUSSION THREAD

**Discussion by zeroku3** [crater-invoice-inc/crater#1286] [General]
I would like negative value taxes to be created. In Spain, self-employed professionals have to record 15% of IPRF. so it is -15% of the total.
Since the app does not allow it to be created, I have put the value -15 in the database, everything works, except that when I generate the PDF that new tax appears as 0, I imagine that because that variable will be an integer value, but I can't find it to change that, can you help me?

**Comment by zeroku3** [comment]
also changed the tax value in the model from integer to float

**Reply by Javiermona** [reply]
Hola yo cree un campo decimal de 20 , 2 en los informes me daba error al final me toca modificar los pdf. La función que formateaba los decimales dependiendo la moneda me daba error a cada momento. Lo cambie por esto {{ number_format( ($tax->total_tax_amount / 100 ) , 2, "," , ".") }}

A ti no te ha dado porblea con tu tipo de dato en la base de dato?

**Comment by zeroku3** [comment]
solved! sorry!
